fish_probes/C_predict_probes.py

```diff
@@ -89,7 +89,53 @@ def find_conserved_regions(seq_sel_clade,k,perc_seq_with_kmer):
     # return
     return kmers_recall,kmers_precision
 
+def get_kmer_sens(seq_sel_clade,kmer):
 
+    #all_strings_kmers = dict()
+    #for s in seq_sel_clade:
+    #   allK,allK_N = find_kmers(seq_sel_clade[s],len(kmer))
+    #   all_strings_kmers[s] = allK
+
+    all_kmers = set()
+    all_kmers.add(kmer)
+    #for s in all_strings_kmers:
+    #   for kmer in all_strings_kmers[s]:
+    #       all_kmers.add(kmer)
+
+    if VERBOSE > 2:
+        #UTIL_log.print_message("Identifed "+str(len(all_kmers))+" unique "+str(k)+"-mers.")
+        UTIL_log.print_message("Calculating sensitivity for k-mer {}".format(kmer))
+    # # now we count how many times it appear
+    count_mers = dict()
+    for kmer in list(all_kmers):
```

Comment:
```suggestion
    for kmer in all_kmers:
```
no need to convert, you can iterate over the set

Reply:
thanks!